src/utils/llm_endpoint.py

```diff
@@ -5,44 +5,38 @@
 
 class LLM:
 
-    def __init__(self, platform: str, model_name: str, key: str):
+    def __init__(self, platform: str, model_name: str):
         self.platform = platform
         self.model_name = model_name
-        self.key = key
-        if self.platform == 'openai':
-            if not self.key:
-                self.key = os.getenv('OPENAI_API_KEY')
-                if not self.key:
-                    raise RuntimeError("'OPENAI_API_KEY' not found via os.getenv" )
-            self.client = OpenAI(
-                api_key=self.key
-            )
+        key = None  # key should be set in environment, not in code.
```

Comment:
let's remove it since it is not used in the same layer.

Reply:
Take a look below. It is used in the initialization of AI client.

Reply:
It is ok to remove here, since it is initialized below.